### PR TITLE
automerge multi comment

### PR DESCRIPTION
**Pull request by talum**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,13 @@
 # repo with collaborator
 
 hi
-
+hello hello
```
Comment (tracylum) [inline README.md:4]: hi

File: README.md [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
 Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur venenatis libero ac est tristique euismod. Nam fermentum sapien eget ipsum tempor, sed suscipit sapien fringilla. Proin id leo ipsum. Sed euismod congue risus id posuere. Ut commodo justo et facilisis tristique. Cras metus nulla, ultrices sit amet venenatis sed, elementum at sem. Pellentesque eu eros lacinia, pulvinar nisi non, ornare arcu. Nam condimentum dictum sapien et tincidunt.
```
Comment (talum) [inline README.md:5]: making a thread!

File: README.md [hunk 3]
```diff
@@ -1,11 +1,13 @@
...
```
Comment (talum) [inline README.md:6]: multi combo

File: README.md [hunk 4]
```diff
@@ -1,11 +1,13 @@
...
 
 Nam dictum, nisl at viverra dignissim, neque odio pellentesque enim, sed pharetra velit tortor eget sapien. Fusce quam libero, suscipit sit amet suscipit in, mattis a lectus. Sed lobortis enim ac ante eleifend, nec malesuada urna bibendum. Praesent ante nulla, aliquet quis lorem quis, tincidunt pellentesque lectus. Aenean in mi vestibulum, imperdiet massa id, vehicula tellus. Fusce maximus elit non vestibulum vulputate. Morbi ut magna magna. Quisque ultricies libero et auctor porttitor. Integer et sollicitudin ante, at feugiat sapien.
```
Comment (talum) [inline README.md:7]: anotha one

**Pull request merged**